generate/avro_test_data/nested.go

```diff
@@ -0,0 +1,14 @@
+package avro_test_data
+
+type Nested struct {
+	FactCheckClaims []struct {
+		Claim          string `json:"claim"`
+		Rating         string `json:"rating"`
+		Author         string `json:"author"`
+		Date           string `json:"date"`
+		AppearanceURLs []struct {
```

Comment:
This file is built with anonymous nested structs. It would be good to have a 2nd file where the nested structs where named. The same schema can generate that if you use the `-nestedStructs=false` flag.